src/Api/Controllers/CollectionsController.cs

```diff
@@ -648,19 +646,16 @@ private async Task<CollectionResponseModel> Post_vNext(Guid orgId, [FromBody] Co
 
         if (!_currentContext.UserId.HasValue || await _currentContext.ProviderUserForOrgAsync(orgId))
         {
-            return new CollectionResponseModel(collection);
+            return CollectionDetailsResponseModel.FromUnassigned(collection);
         }
```

Comment:
I'm not sure if this is a bug - but you can be a providerUser for an org AND a member. It should probably check that they're not a member and are a provider.

Reply:
Good catch, fixed in https://github.com/bitwarden/server/pull/3793/commits/1a4047b8db2f1a5db2ed40754c500560dc053c45